odh/overlays/moc/jupyterhub/pvcs/secret-generator.yaml

```diff
@@ -4,6 +4,7 @@ kind: ksops
 metadata:
   name: jupyterhub-pvc-secret-generator
 files:
-  - ./pvcs/user-mcli-pvc.enc.yaml
-  - ./pvcs/user-hvee-pvc.enc.yaml
   - ./pvcs/user-cdol-pvc.enc.yaml
+  - ./pvcs/user-hvee-pvc.enc.yaml
+  - ./pvcs/user-mcli-pvc.enc.yaml
+  - ./pvcs/user-oc-pvc.enc.yaml
```

Comment:
Is there a reason the file name is not `user-ocha-pvc...`?

Reply:
@4n4nd  that could be because this PVC is associated with my personal email ID `oc@bu.edu`, and this PVC increase on my personal account unblocks my work as the JH account associated with my main account `ochatter@redhat.com` is unable to spawn.

Reply:
@4n4nd as @oindrillac described. The username doesn't have enough letters to do 4 letter resource name. I don't want to introduce conflict/confusion with the possible `ocha` resource for `@redhat` account.

Reply:
this makes sense :+1: 
/lgtm